private/prepare_mesh_fittemplate.m

```diff
@@ -0,0 +1,48 @@
+function M = prepare_mesh_fittemplate(headshape,template)
+
+% PREPARE_MESH_FITTEMPLATE computes an affine transformation matrix between 2 point clouds 
+%
+% This function relies on cpd toolbox found in the external/cpd folder
```

Comment:
Please provide a link (URL) to the toolbox.

Reply:
Added the URL